### PR TITLE
Add "wiitl" as a trigger for labstaff check

### PR DESCRIPTION
**Pull request by NotRyan**
\<fydai> petition to create a shorter command for that so it's easier for me to type while driving

### CODE REVIEW

**Comment by cg505** [comment]
what is the `(?i)`?

**Comment by redplanks** [comment]
Can't it be "!lab"?

**Comment by cg505** [comment]
`!` is harder to type but that's probably also fine

**Comment by ethanhs** [comment]
The original point is to make it easy to type on the phone. On most keyboards you have to change state to reach the `!` key so I would prefer the current implementation.

**Comment by ethanhs** [comment]
@cg505 the `(?i)` is a regex flag to tell it to ignore case. Ctrl+F for `aiLmsux` in https://docs.python.org/3/library/re.html#regular-expression-syntax.

File: ircbot/plugin/lab.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 def register(bot):
     bot.listen(r'is ([a-z]+) in the lab', in_lab, require_mention=True)
     bot.listen(r"(who is|who's) in the lab", who_is_in_lab, require_mention=True)
+    bot.listen(r"(?i)wiitl", who_is_in_lab)
```
Comment (ethanhs) [inline ircbot/plugin/lab.py]: please change this to single quotes so the lint passes. (The listen above uses double quotes do the apostrophe).

**Comment by NotRyan** [comment]
> what is the `(?i)`?
makes the regex case insensitive

**Comment by ethanhs** [comment]
Thank you @NotRyan! I'm sure fydai and others will appreciate this change.

**Pull request merged**